### ISSUE TITLE
Cannot delete files in web editor

### ISSUE THREAD

**Issue by PoolloverNathan** [godotengine/godot#70975]
### Godot version

v3.5.1.stable.official [6fed1ffa3] on web

### System information

Chrome 107.0.5304.110 (Official Build) (64-bit) on ChromeOS 15117.111.0 (Official Build) stable-channel octopus

### Issue description

When using the web editor, it is impossible to delete files. This makes the quick load screen cluttered, wastes disk space, and forces me to resort to a "TRASH" folder.

### Steps to reproduce

1. Create a project on the web editor
2. Create a file
3. Delete it

### Minimal reproduction project

N/A

### Temporary workaround

Creating a trash folder and putting any files that you would delete in there seems to work, as long as you clear it occasionally. You will also have to change the project folder every time you import as a side-effect of this.

**Comment by MeBadDev** [comment]
i don't think we can do anything

**Comment by Calinou** [comment]
> i don't think we can do anything, website can't delete file

It's definitely possible to delete files within a virtual filesystem exposed by browsers.

**Comment by belzecue** [comment]
Just noting that if you can move a file or folder in the web editor (which you can), that is effectively deleting the original file or folder, which suggests it should be normal to delete from the VFS.

**Comment by PoolloverNathan** [comment]
Doing some research, it appears that idbfs has a `unlink` method. I'll make a fork for debugging and potentially fixing this.

**Comment by PoolloverNathan** [comment]
Fixed in/before v4.0.beta9.official [`e780dc332`]

**Comment by PoolloverNathan** [comment]
Seems to be back in/before v4.0.rc3.official [7e79aead9]

**Comment by radlotus** [comment]
Can this be fixed in the 3.x branch?

**Comment by RandomComputerNerd** [comment]
I also have this issue. It is incredibly annoying and I keep having to completely remake projects or pain-stakingly move scenes to a new projects because of this.

**Comment by Calinou** [comment]
> Can this be fixed in the 3.x branch?

Yes, if a solution is found in 4.x. We don't know how to resolve this yet :slightly_smiling_face:

**Comment by aarvinr** [comment]
**Workaround:** If you need to delete a file in your project, exit back to the loader and open the modal for creating a new project. Then, open the panel for choosing the location of your project, and you can delete anything from in there:
![image](https://github.com/godotengine/godot/assets/144612896/c73b18ab-d30f-44bd-834c-4a0657f70c65)

**Comment by ArtyIF** [comment]
This issue also happens on exported web builds, at least in Godot 4.4-dev7. Files in `res://` come back after the game is restarted
